app/parse.py

```diff
@@ -14,5 +17,54 @@ class Course:
     course_type: CourseType
 
 
+BASE_URL = "https://mate.academy/"
+COURSE_FIELDS = [field.name for field in fields(Course)]
+
+
+def get_single_course(
+    course_soup: BeautifulSoup,
+    course_type: CourseType
+) -> Course:
+    return Course(
+        name=course_soup.select_one("a.mb-16").text,
+        short_description=course_soup.select_one(
+            "p.typography_landingMainText__Ux18x"
+        ).text,
+        course_type=course_type
+    )
+
+
 def get_all_courses() -> list[Course]:
-    pass
+    courses = []
+    page = requests.get(BASE_URL).content
+    soup = BeautifulSoup(page, "html.parser")
+
+    full_time_courses = soup.select(
```

Comment:
try to avoid repeating, do itlike this

```
    course_types = [
        ("#full-time section.CourseCard_cardContainer__7_4lK", CourseType.FULL_TIME),
        ("#part-time section.CourseCard_cardContainer__7_4lK", CourseType.PART_TIME)
    ]

    for course_selector, course_type in course_types:
        courses.extend([
            get_single_course(course_soup, course_type)
            for course_soup in soup.select(course_selector)
        ])
```